boards/lilygo/ttgo_lora32/doc/index.rst

```diff
@@ -178,7 +178,7 @@ Build and flash applications as usual (see :ref:`build_an_application` and
    :board: ttgo_lora32/esp32/procpu
    :goals: build
 
-The usual ``flash`` target will work with the ``ttgo_lora32`` board
+The usual ``flash`` target will work with the ``ttgo_lora32`` board target
 configuration. Here is an example for the :zephyr:code-sample:`hello_world`
```

Comment:
-configuration

Reply:
Sorry, I'm unsure when to use target or target configuration.
Fixed.